xml/System.DirectoryServices.AccountManagement/PrincipalValueCollection`1.xml

```diff
@@ -902,7 +954,7 @@
       <Docs>
         <param name="index">An integer.</param>
         <summary>Gets the object at the specified index in the collection.</summary>
-        <value>To be added.</value>
+        <value>The element at the specified index.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[
## Remarks
This member is an explicit interface member implementation. It can be used only when the <xref:System.DirectoryServices.AccountManagement.PrincipalValueCollection`1> instance is cast to an <xref:System.Collections.IList> interface.
          ]]></format>
        </remarks>
```